### PR TITLE
docs: Update 2023-06-11-databend-weekly-97

### PR DESCRIPTION
**Pull request by soyeric128**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: website/blog/2023-06-11-databend-weekly-97.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Stay connected with the latest news about Databend.
 
 ### Column Position
```
Comment (soyeric128) [inline website/blog/2023-06-11-databend-weekly-97.md:37]: I am not sure if I understand this feature correctly. Please review this part carefully....

Comment (soyeric128) [reply website/blog/2023-06-11-databend-weekly-97.md:37]: @PsiACE

File: website/blog/2023-06-11-databend-weekly-97.md [hunk 2]
```diff
@@ -52,15 +52,23 @@ select $1, $2, a, b from t1;
...
 
-Please note that when using `select` for NDJson, only `$1` is allowed. This stands for the entire line and has a variant type. On the other hand, when copying into `my_table` from `@my_stage`, matching is done by name of top-level fields.
+It is important to note that when using the SELECT statement for NDJSON in Databend, only $1 is allowed, representing the entire row and having the data type variant. 
+
+```sql
+-- Select the entire row using column position:
+SELECT $1 FROM @my_stage
+
+--Select a specific field named "a" using column position:
+SELECT $1:a FROM @my_stage
```
Comment (youngsofun) [inline website/blog/2023-06-11-databend-weekly-97.md:70]: need to specify file_format.
either when `create stage` or when select/copy, note copy with transform and select has different way to do it:

```
SELECT $1:a FROM @my_stage (file_format=>'ndjson')
copy into my_table from (select $1 from @my_stage t) FILE_FORMAT = (type = NDJSON)
```

Comment (youngsofun) [reply website/blog/2023-06-11-databend-weekly-97.md:70]: rest lgtm

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**